nuxt.config.ts

```diff
@@ -55,9 +55,12 @@ export default defineNuxtConfig({
     'nuxt-svgo',
     '@pinia/nuxt',
     '@pinia-plugin-persistedstate/nuxt',
-    //'vue-recaptcha/nuxt',
-    '@zadigetvoltaire/nuxt-gtm'
+    '@zadigetvoltaire/nuxt-gtm',
+    '@nuxtjs/turnstile'
   ],
+  turnstile: {
+    siteKey: process.env.NUXT_PUBLIC_TURNSTILE_SITE_KEY || '0x4AAAAAAATLCwNJ5HNQWRsX'
```

Comment:
Why is this a site key or secret key? Also, shouldn't this use NUXT_PUBLIC_TURNSTILE_SITE_KEY instead of hardcoded here?

Reply:
It is the sitekey only, I thought this one does not need to be secret... you can override it with the env variable. the secret needs to be in an env variable

Reply:
ah i see, ok that makes sense